spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/actions/DeleteOrphanFilesSparkAction.java

```diff
@@ -143,6 +148,27 @@ public DeleteOrphanFilesSparkAction executeDeleteWith(ExecutorService executorSe
     return this;
   }
 
+  @Override
+  public DeleteOrphanFiles newPrefixMismatchMode(PrefixMismatchMode mismatchMode) {
+    this.prefixMismatchMode = mismatchMode;
+    return this;
+  }
+
+  @Override
+  public DeleteOrphanFiles newEqualSchemes(Map<String, String> schemes) {
```

Comment:
We should return `DeleteOrphanFilesSparkAction` in all new methods in the implementation class.

Reply:
done